### ISSUE TITLE
Increase visibility of games requiring scoring confirmation

### ISSUE THREAD

**Issue by yewang** [online-go/online-go.com#700]
### Summary

Currently, it is quite easy to overlook a correspondence game that has entered the scoring (stone removal) phase, where you are required to mark/confirm the life/death status of the stones. 

Suggestion: increase the visibility of these games with the following changes
- Send an email notification when marking or confirmation of stones are required.
- Add these games to the rotation of games where its your move to play (as accessed by auto-advance or clicking the number in the top-right).

### Environment

Website behavior appears to affect all browsers. Seen on Chrome and Firefox on Linux, Windows, and Android.

### Steps To Reproduce:

1. Enter scoring (stone removal) phase of a correspondence game, but leave the stone status unconfirmed. Alternatively, you can confirm stone status first, but then have your confirmation cleared by your opponent changing the stone status.
2. Cycle through games with your move to play (by auto-advancing from another game or clicking on the advance to next game button in the top-right).
3. Check email for notifications (if scoring phase entered by opponent making second pass while you were away from the site).

### Additional Information:

This issue is described further as the second item in this forum post:
https://forums.online-go.com/t/two-suggestions-related-to-correspondence-games/17979

**Comment by flovo** [comment]
> Add these games to the rotation of games where its your move to play (as accessed by auto-advance or clicking the number in the top-right).

These games should already be in the rotation (see line 305-314 below). 

For some reason, directly after connecting to `'notification/connect'`, we get notifications for all games in phase 'play', but not for games in phase 'stone removal'. On the other hand, if we are already connected, we get notifications for games which enter 'stone removal' as well. These games are added to the rotation as expected.

https://github.com/online-go/online-go.com/blob/93c70b6db82c66c446db7600014043207834e6be/src/components/Notifications/Notifications.tsx#L305-L314

**Comment by BHydden** [comment]
This might be a stupid question... but when you look at the API is the game phase "stone removal" or "stone_removal"?

**Comment by flovo** [comment]
<details><summary> 'phase': 'stone removal'</summary>

{'id': 5882, 'phase': 'play', 'name': 'Analysis Disabled', 'player_to_move': 815, 'width': 19, 'height': 19, 'move_number': 0, 'paused': 0, 'private': False, 'black': {'username': 'flovo_test', 'id': 815, 'rank': 17.749501175185603, 'professional': False, 'accepted': False, 'ratings': {'overall': {'rating': 1500, 'deviation': 350, 'volatility': 0.06}}}, 'white': {'username': 'GnuGo', 'id': 3, 'rank': 17.749501175185603, 'professional': False, 'accepted': False, 'ratings': {'overall': {'rating': 1500, 'deviation': 350, 'volatility': 0.06, 'games_played': 0}}}, 'time_per_move': 999999}
</details>

**Comment by flovo** [comment]
We could work around it by using https://online-go.com/api/v1/ui/overview to initialisize the turn-indicator, but this would introduce raise conditions.

**Comment by BHydden** [comment]
Yeah probably best for anoek to fix whatever is broken on the back end that isn't pushing score phase games into the notification like the others than to hack the API into the equation :P I don't think anoek would like that solution... just a headache for tomorrow ;)

**Comment by flovo** [comment]
Guess why I haven't created a patch yet 😉

**Comment by BHydden** [comment]
It could be the only reason, you've fixed everything else on the site already ;)

**Comment by koshogs** [comment]
Copied from this forum post: https://forums.online-go.com/t/why-arent-games-ready-to-count-showing-up-as-active-games/34159/10

After running a test on [Beta](https://beta.online-go.com) I discovered this: If you leave the game page without accepting the score or if your opponent modifies the score (requiring you to reaccept) then the game correctly displays as active. 

However; if you go to the home page and refresh(f5) then the game reliably disappears from the active counter even if it still requires your scoring attention for whatever reason. This is highly likely to affect correspondence games, assuming that closing and reopening the page has the same effect as refreshing.

Before and after refreshing:

![ActiveCycle Bug](https://user-images.githubusercontent.com/44488770/107730209-ee6a1500-6d46-11eb-900e-85a146507758.png)

**Comment by anoek** [comment]
Not stale and shall be fixed soon finally.

**Comment by anoek** [comment]
Ok this is finally fixed and deployed.
